File: atcoder/abc190A_very_very_primitive_game.nim
# uninhm
# https://atcoder.jp/contests/abc190/tasks/abc190_a
# implementation

import strutils, sequtils

var a, b, c: int
(a, b, c) = stdin.readLine.split.map(parseInt)

if a > b:
  echo "Takahashi"
elif b > a:
  echo "Aoki"
else:
  if c == 0:
    echo "Aoki"
  else:
    echo "Takahashi"

File: atcoder/abc190A_very_very_primitive_game.hs
-- user: nel_tu_
-- path: https://atcoder.jp/contests/abc190/tasks/abc190_a
-- tags: implementation

import Data.Bool ( bool )

solve [a, b, c] = bool "Aoki" "Takahashi" $ a - b + c > 0

main = interact $ solve . map read . words

